.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ commands:
           condition: << parameters.linux >>
           steps:
             - run: sudo apt-get update
-            - run: sudo apt-get install ocl-icd-opencl-dev
```

Comment:
Why is this needed now? Is it due to the FFI upgrade?

Reply:
yea :(

Reply:
turned out didn't need ffi update so removed

Reply:
just kidding now I remember I had to do this in order to compile the FFI for M1 :(